proposals/0000-reduce-fragmentation.md

```diff
@@ -0,0 +1,1047 @@
+
+* [ ] [`blockquote`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote).
+* [ ] [`li`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li).
+* [ ] [`ol`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol).
+* [ ] [`ul`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul).
+* [ ] [`p`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p).
+* [ ] [`pre`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre).
+
+## Inline text
+
+Inline text semantics define the meaning, structure, or style of a word, line, or any arbitrary piece of text.
+
+* [ ] [`a`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a).
+* [ ] [`b`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b).
+* [ ] [`bdi`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdi).
+* [ ] [`bdo`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdo).
+* [ ] [`code`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code).
+* [ ] [`em`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em).
+* [ ] [`span`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span) is equivalent to `Text` with `display:inline` layout.
+* [ ] [`strong`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong).
+* [ ] [`sub`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/sub).
+* [ ] [`sup`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/sup).
+
+## Media
+
+* [ ] [`img`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img) is similar to `Image` without advanced loader configuration.
+
+## Forms
+
+Elements which can be used together to create forms which the user can fill out and submit. There's a great deal of further information about this available in the [HTML forms guide](https://developer.mozilla.org/en-US/docs/Learn/Forms).
+
+* [ ] [`button`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button).
+* [ ] [`input`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) is equivalent to `TextInput` with `inputMode` corresponding to `type`. Expanding input type support to number, datetime, color, etc., resolves proposal [#510](https://github.com/react-native-community/discussions-and-proposals/issues/510).
+* [ ] [`label`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label).
+* [ ] [`optgroup`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup).
+* [ ] [`option`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option).
+* [ ] [`progress`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress).
+* [ ] [`select`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select).
+* [ ] [`textarea`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) is equivalent to `TextInput` with `multiline={true}`.
+
```

Comment:
`<table>` and all associated elements seem important too given how prominent they are in markdown and MDX on web.

Reply:
Do OSS stakeholders really want tables and have an idea of how to implement that in React Native?